src/Buttons/Button.jsx

```diff
@@ -30,7 +30,14 @@ export default function Button({
 Button.propTypes = {
   children: PropTypes.node.isRequired,
   onClick: PropTypes.func,
-  type: PropTypes.string,
+  type: PropTypes.oneOf([
+    "primary",
+    "secondary",
+    "tertiary",
+    "quaternary",
+    "white",
+    "grey"
```

Comment:
I think there might be some confusion here due to the current designs, lets discuss in person as I don't think there was meant to be a grey button, just a disabled state for the previous buttons

Also I dislike that we now have button types that are colours and a button types that are functional mixed, it doesn't make sense.

In fact I would say we rather just style the quaternary button to fit the "white" button use case as it's not use anywhere from what I can see, and then we can keep the number of different button styles down to 4 instead of 6

Reply:
agree.